dhis-2/dhis-web-api/src/main/java/org/hisp/dhis/webapi/controller/ResourceTableController.java

```diff
@@ -106,9 +107,13 @@ public WebMessage analytics(
             skipTableTypes.add( AnalyticsTableType.OWNERSHIP );
         }
 
+        if ( skipOrgUnitOwnership )
```

Comment:
Now that we have a proper way of skipping the ownership tables, please remove these lines just above:

        if ( skipEvents && skipEnrollment )
        {
            skipTableTypes.add( AnalyticsTableType.OWNERSHIP );
        }

This will allow users to build the ownership tables only at a different time if they want to do that, and then generate the events and enrollments tables at a different time.

Otherwise, looks great, thanks!!!!!

Reply:
Done